ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -50,6 +50,19 @@
                                       {::t-service/sub-type sub-type
                                        ::t-service/type (service-type-from-sub-type sub-type)})))))
 
+(define-event ShowBrokeringServiceDialog []
+  {}
+  (assoc-in app [:transport-service :show-brokering-service-dialog?] true))
+
+(define-event SelectBrokeringService [select-type]
+  {}
+  (let [type (get-in app [:transport-service ::t-service/type])
+        type-key (t-service/service-key-by-type type)]
+  (cond-> app
+          (when (not select-type)) (assoc :show-brokering-service-dialog? false)
```

Comment:
In `cond` the first arg is a boolean true/false whether the next element should be evaluated or not. While `when` probably works because it returns either a alue or nil, it might be more logical to simply use a boolean evaluation  i.e. ```(not select-type)´´´

Reply:
Really good point. thanx!